src/main/java/com/uid2/admin/vertx/service/SharingService.java

```diff
@@ -75,43 +75,81 @@ private void handleSetKeyset(RoutingContext rc) {
                 keysetProvider.loadContent();
                 siteProvider.loadContent();
             } catch (Exception e) {
-                LOGGER.error("Failed to load key acls");
+                LOGGER.error("Failed to load keysets");
```

Comment:
We should log the exception. Consider writing a function to both log the exception and rc.fail(500), since those go hand in hand.

Reply:
Added this as ResponseUtil#errorInternal